start-services.sh

```diff
@@ -68,12 +71,14 @@ do
 
     qdb_start "${ARGS_INSECURE}" ${CONSOLE_LOG_INSECURE} ${CONSOLE_ERR_LOG_INSECURE}
 
-    echo "Cluster secure:"
-    ARGS_SECURE="--id ${NODE_ID} -a ${THIS_URI_SECURE} -r ${THIS_DATA_DIR_SECURE} -l ${THIS_LOG_DIR_SECURE} --enable-performance-profiling  --with-firehose \$qdb.firehose --security=true --cluster-private-file=${CLUSTER_PRIVATE_KEY} --user-list=${USER_LIST}"
-    if [[ -f ${CONFIG_SECURE} ]]; then
-        ARGS_SECURE="${ARGS_SECURE} -c ${CONFIG_SECURE}"
+    if [ ${QDB_DISABLE_SECURE_CLUSTER} ] ; then
```

Comment:
Is this condition correct?

I tend to prefer `QDB_ENABLE_SECURE_CLUSTER` and it being ON by default.

Reply:
No opinion on that. I like both.
Fixed the condition.